### ISSUE TITLE
Final Project Peer Review

### ISSUE THREAD

**Issue by jiangckevin** [yashmundra13/ORIE4741#18]
This project is about predicting the level of water contamination at a fracking site given certain features, and the data set comes from well sampling reports reported by the EPA in the years 2014,2015,2016.

(+s)
	It was very easy to follow the report, as there were informative headings and an intuitive layout behind the entire report.
	I appreciated how you tried models/techniques not used in class, and I particularly enjoyed reading these sections.
	You addressed all bullet points on the rubric and made it clear that you answered everything that was asked.

(-s)
	Perhaps you should consider adding an explanation for Neural Networks and Ablation study, as you simply stated the hyperparameters and results. Although interested readers should be somewhat familiar with these concepts, I would give thorough background information.
	I would have liked to see more visualizations aside from a very high-level geographical picture of the fracking data and the cross validation results. It would make the report more interesting.
	Some tables are either unclear or tell unnecessary information -- what exactly is "distance_20_nearest" and why should I care if Latitude Fracking and Longitude Fracking have the some accuracy in the ablation study? More analysis would be appreciated.

I enjoyed reading your report! Overall, nicely done.
